ordino/storage.py

```diff
@@ -30,8 +30,9 @@ def get_namedsets():
     description = request.values.get('description', '')
     sub_type_key = request.values.get('subTypeKey', '')
     sub_type_value = request.values.get('subTypeValue', '')
+    type = int(request.values.get('type', ''))
```

Comment:
The ENamedSetType, which is added inside the saveNamedSet function
I needed the type inside the return value